incubator/jenkins/templates/config.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
```

Comment:
In order to preconfigure the kubernetes plugin we need to populate the config.xml with some templated variables.